api/src/controllers/homepage.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * This module contains handlers for the "homepage" route.
+ * @module homepage
+ */
+const Homepage = require('../models/homepage.model');
+const { fillErrorObject } = require('../middleware/error');
+
+/**
+ * Get homepage content on /homepage/:team_id
+ * @param {*} req
+ * @param {*} res
+ * @returns
+ */
+async function getHomepage(req, res, next) {
+  const { team_id } = req.params;
+  try {
+    const foundHomepage = await Homepage.findOne({ teamId: team_id });
+    if (foundHomepage) {
+      return res.status(200).json(foundHomepage);
+    }
+    const emptyHomepage = { teamId: team_id, aboutUs: [''] };
+    return res.status(200).json(emptyHomepage);
+  } catch (err) {
+    next(fillErrorObject(500, 'Server error', [err.errors]));
```

Comment:
minor: we should prefix this statement with `return` to guard against executing any introduced code below.

Think about what happens if we add code after L25; we'd call `next()` **AND** we'd execute any other code, which we don't want because we want to `return` early if there's an error. I'm calling this out because we've encountered bugs exactly due to this before

Reply:
Thanks for pointing that out! Will refactor them!